tests/dashbio_demos/app_manhattan_plot.py

```diff
@@ -40,94 +33,89 @@ def description():
-                        className='row mhp-text mhp-intro',
-                        children="Visualize genome wide association studies "
-                                 "(GWAS) with efficient manhattan plots. "
-                                 "Using WebGL under the hood, interactively "
-                                 "explore hundred of thousands of points at "
-                                 "once or individually hover over them.",
-                    ),
-                    html.Div(
-                        className='mhp-horizontal-style mph-control-div',
-                        id='mhp-slider-genome-div',
-                        children=[
-                            html.H5(
-                                "Threshold value (red line)",
-                                className='mhp-text',
-                            ),
-                            html.Div(
-
-                                className='mhp-slider-div',
-                                children=dcc.Slider(
-                                    id='mhp-slider-genome',
-                                    vertical=False,
-                                    updatemode='mouseup',
-                                    max=9,
-                                    min=1,
-                                    value=7,
-                                    marks={
-                                        i + 1: '{}'.format(i + 1)
-                                        for i in range(9)
-                                    },
-                                    step=0.05
-                                ),
-                            )
-                        ]
-                    ),
-                    html.Div(
-                        className='mhp-horizontal-style mph-control-div',
-                        id='mhp-slider-indic-div',
-                        children=[
-                            html.H5(
+    return html.Div(id='mhp-page-content', children=[
+        html.Div(
+            id='mhp-graph-div',
+            children=dcc.Graph(
+                figure=fig,
+                style={
+                    'bgcolor': 'rgba(0,0,0,0)'
+                },
+                id='mhp-graph',
+                config={'scrollZoom': True},
+            )
+        ),
+
+        html.Div(id='manhattan-control-tabs', children=[
+            dcc.Tabs(id='manhattan-tabs', children=[
+                dcc.Tab(
+                    label='About',
+                    value='what-is',
+                    children=html.Div(className='manhattan-tab', children=[
+                        html.H4('What is Manhattan Plot?'),
+                        html.P("ManhattanPlot allows you to visualize genome "
+                               "wide association studies (GWAS) efficiently. "
+                               "Using WebGL under the hood, you can interactively "
+                               "explore hundred of thousands of points at once, or"
```

Comment:
```suggestion
                               "explore hundreds of thousands of points at once, or"
```
I think the description is a little misleading... it's not like one can explore hundreds of thousands of points *at once* (our human brain cannot do it); I guess what you mean is that the user can explore their data at both a global level (because hundreds of thousands of data points are represented) and a micro level (because hovering over a data point lets them view the individual info for this point). So I think it needs a bit of rephrasing.

Reply:
@shammamah Great rewrite (d4eecdbb775996)!